src/Mono.Android/Mono.Android.csproj

```diff
@@ -348,7 +348,9 @@
     <Compile Include="System.Drawing/SizeFConverter.cs" />
     <Compile Include="System.IO\AndroidExtensions.cs" />
     <Compile Include="System.Linq\Extensions.cs" />
-    <Compile Include="Xamarin.Android.Net\AndroidClientHandler.cs" />
+    <Compile Condition=" '$(TargetFramework)' == 'net6.0' " Include="Xamarin.Android.Net\AndroidClientHandler.cs" />
```

Comment:
Problem here is that this will require updates for `net7.0`, `net8.0`, etc.

We should instead "reverse" the logic and use `monoandroid10` as control mechanism.

Additionally, we should fix: https://github.com/xamarin/xamarin-android/blob/57636c8199efe724d1875f05d6b68051dfbcec02/src/Mono.Android/Mono.Android.csproj#L45

to instead use:

```xml
  <PropertyGroup Condition=" '$(TargetFramework)' != 'monoandroid10' ">
```